helm/idol-qms/resources/qms-agentstore.cfg

```
@@ -0,0 +1,235 @@
+[License]
+LicenseServerHost={{ .Values.singleAgentstore.licenseServerHostname }}
+LicenseServerACIPort={{ .Values.singleAgentstore.licenseServerPort | int }}
+LicenseServerTimeout=60000
+LicenseServerRetries=1
+
+//--------------------------- Authorization Roles ----------------------------//
+[AuthorizationRoles]
+0=AdminRole
+1=QueryRole
+2=IndexRole
+
+[AdminRole]
+StandardRoles=admin,servicecontrol,query,servicestatus
+Clients=*.*.*.*
+//SSLIdentities=
+//GSSPrincipals=
+
+[QueryRole]
+StandardRoles=query,servicestatus
+Clients=*.*.*.*
+//SSLIdentities=
+//GSSPrincipals=
+
+[IndexRole]
+StandardRoles=index
+Actions=getpid,getstatus,indexergetstatus
+ServiceActions=getstatusinfo
+Clients=*.*.*.*
+//SSLIdentities=
+//GSSPrincipals=
+
+[SERVICE]
+ServicePort={{ .Values.singleAgentstore.servicePort | int }}
+Access-Control-Allow-Origin=http://localhost:9100
```

```suggestion
Access-Control-Allow-Origin=http://localhost:{{ .Values.singleAgentstore.aciPort | int }}
```
This default should depend on aci port configuration